services/libvirt.nix

```diff
@@ -0,0 +1,6 @@
+{
+  virtualisation.libvirtd = {
```

Comment:
any way to configure the vms in nix?

Reply:
It doesn't appear to. I guess you could embed the XML config in here, but that would look dreadful.

Reply:
Also, it would tie them to the lifecycle of motherlode which I don't really want to do. Managing through virtual machine manager is the preferred way to interact with the vms.